types/googlemaps/index.d.ts

```diff
@@ -2476,19 +2476,22 @@ declare namespace google.maps {
         }
 
         export interface OpeningHours {
-            open_now: boolean,
-            periods: OpeningPeriod[],
-            weekday_text: string[]
+            open_now: boolean;
+            periods: OpeningPeriod[];
+            weekday_text: string[];
         }
 
         export interface OpeningPeriod {
-            open: OpeningHoursTime,
-            close?: OpeningHoursTime
+            open: OpeningHoursTime;
+            close?: OpeningHoursTime;
         }
 
         export interface OpeningHoursTime {
-            day: number,
-            time: string
+            day: number;
+            hours: number;
+            minutes: number;
+            nextDate: number;
```

Comment:
Is `nextDate` correct as a `number`?

Reply:
@martincostello it is in fact a number
![2017-09-22 12 35 47](https://user-images.githubusercontent.com/2039539/30761339-b5278b7c-9f92-11e7-80ea-4d507074f9f2.png)

Reply:
Ok cool. Wasn’t obvious from their naming it’d be an epoch.